torch/_C/__init__.pyi.in

```diff
@@ -133,18 +130,18 @@ class Stream:
     device: _device  # The device of the stream
 
     @overload
-    def __new__(self, device: Optional[DeviceLikeType] = None, *, priority: _int = 0) -> Stream: ...
+    def __new__(cls, device: Optional[DeviceLikeType] = None, *, priority: _int = 0) -> Self: ...
     @overload
-    def __new__(self, stream_id: _int, device_index: _int, device_type: _int, *, priority: _int = 0) -> Stream: ...
+    def __new__(cls, stream_id: _int, device_index: _int, device_type: _int, *, priority: _int = 0) -> Self: ...
     def query(self) -> _bool: ...
     def synchronize(self) -> None: ...
-    def wait_event(self, event: Event) -> None: ...
-    def wait_stream(self, other: Stream) -> None: ...
-    def record_event(self, event: Optional[Event] = None) -> Event: ...
+    def wait_event(self, event: "Event") -> None: ...
```

Comment:
You don't need quotes in a `.pyi` stub file.

Reply:
Event is defined later, for circular reference.

Reply:
We are using `mypy` instead of `pyright`. Quotes also violates `flake8-pyi` (`ruff PYI`) rule [quoted-annotation-in-stub (PYI020)](https://docs.astral.sh/ruff/rules/quoted-annotation-in-stub/#quoted-annotation-in-stub-pyi020).

Reply:
Thank you for pointing it out, the changes are reverted.